### DISCUSSION TITLE
Files in scope

### DISCUSSION THREAD

**Discussion by 0x-codeX** [Cyfrin/security-and-auditing-full-course-s23#187] [Q&A] [answered]
I am 60% in this course and it has been interesting, I have been checking out some projects and I know several things to do to start auditing almost any project but I would like to complete the course before I do that, I do not wanna drop off halfway thinking I know it all.
I just wanted to be clear on something though, ¨File(s) in scope¨ are the the contracts we are supposed to audit in an audit contest?

**Comment by EngrPips** [comment, marked as answer]
Correct!

**Reply by 0x-codeX** [reply]
Thank you.

**Comment by EngrPips** [comment]
You are welcome.
